src/synapse/workload.py

```diff
@@ -69,14 +69,28 @@ class ExecResult(typing.NamedTuple):
 
 
 def check_ready() -> typing.Dict:
-    """Return the Synapse container check.
+    """Return the Synapse container ready check.
 
     Returns:
         Dict: check object converted to its dict representation.
     """
     check = Check(CHECK_READY_NAME)
     check.override = "replace"
     check.level = "ready"
+    check.http = {"url": f"http://localhost:{SYNAPSE_PORT}/_synapse/admin/v1/server_version"}
+    # _CheckDict cannot be imported
+    return check.to_dict()  # type: ignore
```

Comment:
@benhoyt any comment on this please?

Reply:
This is not true anymore. I made `CheckDict` public in ops 2.4 in [as part of this PR](https://github.com/canonical/operator/pull/942/files#diff-d74b54ce28b898e8febb0f33f24abc9bf91bb7793c51ae716bf9a9816d90104d). It's not imported to the top level of `ops`, but it's accessible as `ops.pebble.CheckDict` (similar with `PlanDict` and so on).

Reply:
FIxed, thanks.